appmanifest/shortcuts-member/shortcuts-member-skip-for-empty-name.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Shortcut test",
+    "scope": "/",
+    "shortcuts": [{
+        "name": "",
+        "short_name": "",
+        "description": "",
+        "url": "",
+        "icons": []
+    }, {
+        "name": "pass",
+        "short_name": "",
+        "description": "",
+        "url": "",
```

Comment:
Empty URL means whatever the scope is, right (based on shortcut URL calculation)?

Reply:
> Set shortcut["url"] to the result of parsing shortcut["url"] using manifest URL as the base URL. If the result is failure, issue a developer warning and continue.

From my understanding, an empty URL would link to the manifest file (see [processing the shortcuts member, step 3](https://www.w3.org/TR/appmanifest/#dfn-processing-the-shortcuts-member)). Not sure if that's cool though. We might want to add a test for this special case.

Reply:
That's developer error, I think. It's like doing the following in CSS:

```
body {
  background-image: url(""); 
}
```

Reply:
As such, should we suggest UAs provide a warning in the console?

Reply:
No, as it's not really an error. However, a UA would be free to show something in dev tools, but it wouldn't be something we would normally put in the spec.